src/sql/statement.rs

```diff
@@ -965,182 +945,135 @@ fn handle_create_source(scx: &StatementContext, stmt: Statement) -> Result<Plan,
                         Some(Value::SingleQuotedString(region)) => region,
                         _ => bail!("Kinesis sources require a `region` option"),
                     };
-                    if !with_options.is_empty() {
-                        bail!(
-                            "Unexpected WITH options: {}",
-                            join(with_options.keys(), ",")
-                        )
-                    }
-
-                    match envelope {
-                        dataflow_types::Envelope::None => {}
-                        dataflow_types::Envelope::Debezium => {
-                            bail!("Debezium-envelope Kinesis sources are not supported")
-                        }
-                    }
-
-                    let (encoding, desc) = match format {
-                        Format::Bytes => (
-                            DataEncoding::Bytes,
-                            RelationDesc::new(
-                                RelationType::new(vec![ColumnType::new(ScalarType::Bytes)]),
-                                vec![Some(String::from("record"))],
-                            ),
-                        ),
-                        _ => bail!("Kinesis sources only support data as bytes"),
-                    };
-
-                    let source = Source {
-                        create_sql: "<filled in later>".into(),
-                        connector: SourceConnector::External {
-                            connector: ExternalSourceConnector::Kinesis(KinesisSourceConnector {
-                                arn: arn.clone(),
-                                access_key,
-                                secret_access_key,
-                                region,
-                            }),
-                            encoding,
-                            envelope: Envelope::None,
-                            consistency: Consistency::RealTime,
-                        },
-                        desc,
-                    };
-                    dbg!(&source);
-
+                    dbg!(&ExternalSourceConnector::Kinesis(KinesisSourceConnector {
+                        arn: arn.clone(),
+                        access_key,
+                        secret_access_key,
+                        region
+                    }));
```

Comment:
`dbg!`

Reply:
this existed already, I just moved it somewhere new